flutter/Makefile

```diff
@@ -29,9 +29,7 @@ _bazel_links_arg=--symlink_prefix ${BAZEL_LINKS_DIR} --experimental_no_product_n
 .PHONY: cpp-ios
 cpp-ios:
 	@# NOTE: add `--copt -g` for debug info (but the resulting library would be 0.5 GiB)
-	bazel \
-		${BAZEL_CACHE_FLAG} \
-		build --config=ios_fat64 -c opt //flutter/cpp/flutter:ios_backend_fw_static
+	bazel ${BAZEL_OUTPUT_ROOT_ARG} build --config=ios_fat64 -c opt //flutter/cpp/flutter:ios_backend_fw_static
```

Comment:
Where will `BAZEL_OUTPUT_ROOT_ARG` be set?

Reply:
It is set in the iOS Jenkins job.
This is the main and only reason we had `BAZEL_CACHE_FLAG`.
The name was confusing to me so I renamed the env.